### PR TITLE
Support different levels of diagnostic messages in APIView

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Added  levels for diagnostic messages and new style for info, warning and Error

### CODE REVIEW

**Comment by praveenkuttappan** [comment]
@JonathanGiles FYI.. I have added support for diagnostic levels in this PR.

File: src/dotnet/APIView/APIViewWeb/Pages/Assemblies/_CodeLine.cshtml [hunk 1]
```diff
@@ -26,7 +26,14 @@
         <td colspan="2">
             @foreach (var lineDiagnostic in Model.Diagnostics)
             {
-                <p>
+                string diagClass = lineDiagnostic.Level switch
+                {
+                    APIView.CodeDiagnosticLevel.Error => "code-diagnostics-error",
```
Comment (pakrym) [inline src/dotnet/APIView/APIViewWeb/Pages/Assemblies/_CodeLine.cshtml:31]: You can remove this line the `                    _ => "code-diagnostics-error"
` would handle it

File: src/dotnet/APIView/APIViewWeb/Pages/Assemblies/_CodeLine.cshtml [hunk 2]
```diff
@@ -26,7 +26,14 @@
...
+                    APIView.CodeDiagnosticLevel.Warning => "code-diagnostics-warn",
+                    APIView.CodeDiagnosticLevel.Info => "code-diagnostics-info",
+                    _ => "code-diagnostics-error"
+                };
+                <p class="@diagClass">
```
Comment (pakrym) [inline src/dotnet/APIView/APIViewWeb/Pages/Assemblies/_CodeLine.cshtml:36]: why do we apply the class here and not to the same element as code-diagnostics

**Review by pakrym — APPROVED** [review]

**Comment by JonathanGiles** [comment]
I have merged #975

File: src/dotnet/APIView/APIViewWeb/Pages/Assemblies/_CodeLine.cshtml (hunk 2 above)
Comment (praveenkuttappan) [reply src/dotnet/APIView/APIViewWeb/Pages/Assemblies/_CodeLine.cshtml:36]: Adding at a root row level will be an issue if there are multiple diag messages with different levels for a single line.

**Pull request merged**